apps/testing-javascript/src/path-to-purchase/pricing.tsx

```diff
@@ -20,8 +20,29 @@ import BuyMoreSeats from './buy-more-seats'
 import Icon from 'components/icons'
 import {useDebounce} from '@skillrecordings/skill-lesson/hooks/use-debounce'
 
-function getFirstPPPCoupon(availableCoupons: any[] = []) {
-  return find(availableCoupons, (coupon) => coupon.type === 'ppp') || false
+type MinimalMerchantCoupon = Omit<
+  MerchantCoupon & {
+    country?: string
+  },
+  'identifier'
+>
```

Comment:
Defines a `MerchantCoupon`-subset type that can be used throughout this file to add a bit more consistency. This works well with what is coming from `usePriceCheck`, `trpc`, etc.